core/domain/feedback_services.py

```diff
@@ -448,7 +656,13 @@ def clear_feedback_message_references(user_id, exploration_id, thread_id):
 def _get_all_recipient_ids(exploration_id, thread_id, author_id):
     """Fetches all valid recipient ids from exploration and thread.
```

Comment:
from the given exploration and thread.

(though I don't understand, from this docstring, how the author_id comes into it?)